.ci/docker/build.sh

```diff
@@ -69,6 +69,9 @@ DOCKERFILE="${OS}/Dockerfile"
 # When using ubuntu - 22.04, start from Ubuntu docker image, instead of nvidia/cuda docker image.
 if [[ "$image" == *cuda* && "$UBUNTU_VERSION" != "22.04" ]]; then
   DOCKERFILE="${OS}-cuda/Dockerfile"
+elif [ "$3" = "ARC" ]; then
+  DOCKERFILE="${OS}-arc/Dockerfile"
+  set -- "${@:1:2}" "${@:4}"
```

Comment:
What does this do? I consider myself a bash expert and I'm a bit confused here.

Reply:
It does 'shift' but we don't need it anymore.